csi.proto

```diff
@@ -70,9 +70,11 @@ message GetSupportedVersionsResponse {
 // Specifies a version in Semantic Version 2.0 format.
 // (http://semver.org/spec/v2.0.0.html)
 message Version {
-  uint32 major = 1;  // This field is REQUIRED.
-  uint32 minor = 2;  // This field is REQUIRED.
-  uint32 patch = 3;  // This field is REQUIRED.
+  // NOTE: Plugin must ensure in returning valid **int** values,
+  // **negative** values must **not** be returned.
```

Comment:
How about: `The value of these fields must not be negative.` for this one.

Reply:
what you suggest is to replace second line in the NOTE, right?

Reply:
Correct, I would suggest replace both lines with this.

Reply:
done